head/core/can_task.cpp

```diff
@@ -53,12 +54,12 @@ static auto can_move_group_handler_left =
 
 /** Handler of device info requests. */
 static auto device_info_handler_right =
-    device_info_handler::DeviceInfoHandler(right_queues, 0);
+    device_info_handler::DeviceInfoHandler(right_queues, VERSION);
```

Comment:
could prolly just use one handler to take care of both the heads.

Reply:
I think we can address this in a follow-up PR.